### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by aquaticdosDev**
dont ask

### CODE REVIEW

**Comment by GhostSlayer** [comment]
i wont ask

**Comment by HappyDarion** [comment]
i also wont ask

**Comment by HappyDarion** [comment]
howa are u guys

**Comment by HappyDarion** [comment]
guys dont leave me alone heere

**Comment by HappyDarion** [comment]
please guys dont leave me here

**Comment by HappyDarion** [comment]
guys?

**Comment by HappyDarion** [comment]
where are you guys?

**Comment by HappyDarion** [comment]
you guys still here?

File: index.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ function ddos(webiste) {
   console.log(`Sent 64 Packets from ${proxy} to https://${webiste}`)
 }
 
+while(true) {
```
Comment (aquaticdosDev) [inline index.js:15]: tihs lin cekhc fi ture

File: index.js [hunk 2]
```diff
@@ -12,6 +12,10 @@ function ddos(webiste) {
...
+console.log("ddosing the improster!!!!!!!!!")
```
Comment (aquaticdosDev) [inline index.js:16]: tihs lin say dos improtst

**Comment by HappyDarion** [comment]
so you guys still here

File: index.js (hunk 2 above)
Comment (HappyDarion) [reply index.js:16]: dos imrots

**Comment by GhostSlayer** [comment]
k

**Pull request merged**